docker-compose-generator/docker-fragments/opt-add-bwt.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
+
+services:
+  bwt:
+    image: shesek/bwt
+    environment:
+      network: mainnet
+      bitcoind-url: http://btcpayserver_bitcoind
+      xpub: <xpub1>,<xpub2>,<xpub3>
```

Comment:
needs to be configurable from the setup

Reply:
The env vars should be uppercase and with an underscore, i.e. `NETWORK` or `BITCOIND_URL`. xpub should also be in plural, `XPUBS`.

Reply:
It would be nice if btcpay could automatically configure bwt with the stores xpubs, in addition to the custom user provided ones.

(Note that not bwt doesn't yet support all of btcpay's derivation schemes, only simple p2pkh/p2sh-p2wpkh/p2wpkh for xpub/ypub/zpub... so no multisig etc. This will change once [descriptor support](https://github.com/shesek/bwt/issues/1) lands in bwt.)

Reply:
> It would be nice if btcpay could automatically configure bwt with the stores xpubs, in addition to the custom user provided ones.
> 
> (Note that not bwt doesn't yet support all of btcpay's derivation schemes, only simple p2pkh/p2sh-p2wpkh/p2wpkh for xpub/ypub/zpub... so no multisig etc. This will change once [descriptor support](https://github.com/shesek/bwt/issues/1) lands in bwt.)

 This is not possible, since btcpay is multi user and multi tenant and would require a bridge to sync all currently used xpubs in BTCPay to be transmitted to bwt and would probably be a security risk

Reply:
Another thing to note is that `--xpub`/`XPUBS` tracks both the external and internal chains (`m/0/i` and `m/1/i`). If you don't want that, you can use `--bare-xpub`/`BARE_XPUBS`.